.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - "greenkeeper/**"
```

Comment:
In reference to [other conversation](https://github.com/probot/create-probot-app/pull/112/files#r368074993), the `test` workflow from octokit/core.js includes the `master` branch. Here, I've removed it as changes merged to master are guaranteed to be successful.

Reply:
good point